library/tasks/ZarrUtils.wdl

```diff
@@ -93,6 +93,8 @@ task OptimusZarrConversion {
   command {
     set -euo pipefail
 
+    [[ ~{counting_mode} == "sc_rna" ]] && expression_data_type_param="exonic"  || expression_data_type_param="whole_transcript"
```

Comment:
This code doesn't seem complete

Reply:
It acts like a ternary operator in C

Reply:
Maybe we should expand it to an if for readability?

Reply:
But `expression_data_type_param` has no value here, other than the text itself. It's not declared as a task input.